specification/storage/resource-manager/Microsoft.Storage/stable/2025-01-01/storage.json

```diff
@@ -3075,6 +3075,10 @@
           "$ref": "#/definitions/ActiveDirectoryProperties",
           "description": "Required if directoryServiceOptions are AD, optional if they are AADKERB."
         },
+        "smbOAuthSettings": {
```

Comment:
Please:
1. add swagger example (or update existing one) to cover this change. (Create/update/get)
2. Is there any region ready to test this new property? Any Feature flag registry needed?
3. Is there any limitation on this new property, like does it work with all kind/sku of storage accounts. Does it work with all options of "directoryServiceOptions"? If so, please add it to the description.